src/scripts/main.js

```diff
@@ -1 +1,62 @@
 'use strict';
+
+function notification(bool, message) {
+  const div = document.createElement('div');
+  const p = document.createElement('p');
+
+  p.innerText = message;
+
+  div.append(p);
+  div.setAttribute('data-qa', 'notification');
+
+  if (bool) {
```

Comment:
```suggestion
  if (messageType) {
```
что такое bool? Давай понятные имена переменным